ManagedCode.Storage.Client/StorageClient.cs

```diff
@@ -126,37 +123,45 @@ public async Task<Result<LocalFile>> DownloadFile(string fileName, string apiUrl
         }
     }
     
-    public async Task<Result> UploadFileInChunks(Stream file, string apiUrl, int chunkSize, CancellationToken cancellationToken)
+    public async Task<Result> UploadLargeFile(Stream file, string apiUrl, CancellationToken cancellationToken)
     {
-        var buffer = new byte[chunkSize];
+        var bufferSize = 4096000; //TODO: chunk size get from config
+        var buffer = new byte[bufferSize];
         int bytesRead;
-        int chunkIndex = 0;
+        int chunkIndex = 1;
+        var fileCRC = Crc32Helper.Calculate(file);
```

Comment:
Can you create crc helper from stream? 
also let’s check performance of this crc calculation